ncard_app/templates/tables/awards.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'events/base.html' %}
+{% load render_table from django_tables2 %}
+{% load querystring from django_tables2 %}
+{% load bootstrap5 %}
+{% load crispy_forms_tags %}
+{% block title %}Awards | NCARD Table{% endblock %}
+
+{% block content%}
+<div class="row mb-4">
+    <div class="col-md-3">
+            <h1 class="ncard-dark-green" >Awards</h1>
+
+    </div>
+
+    {% if filter %}
+        <div class="col-md-3" style="align-self: center;">
+            <form action="" method="get" class="form form-inline">
+                {% bootstrap_form filter.form layout='inline'  %}
+            </form>
+        </div>
+    {% endif %}
+    <div class="col-md-4"></div>
+    <div class="col-md-2">
+        <a href="{% url 'add-award' %}">
+            <button class="ncard_CTA">ADD</button>
+        </a>
+    </div>
+</div>
+
+<div id="ncard_table">
+     {% render_table table 'django_tables2/bootstrap4.html' %}
+</div>
+
+    <div class="export_csv"><button class="btn btn-primary mb-3" ><a class="export_href"   href="{% querystring '_export'='csv' %}">export</a></button></div>
```

Comment:
If you put a clickable link inside a button, the text of the button will act like a link but the rest of the button won't, which is unexpected. Instead, style the link itself so it looks like a button.

```suggestion
    <div class="export_csv"><a class="btn btn-primary mb-3 export_href" href="{% querystring '_export'='csv' %}">export</a></div>
```